Local site is unstyled / style module not found

## Expected Behavior

Following the setup steps should result in a working, styled 18F site.
![Screenshot 2024-04-11 at 1 53 43 PM](https://github.com/18F/18f.gsa.gov/assets/1887182/fe0f84bc-cccb-470b-8e26-d73c09b335ba)

## Actual Behavior

Following the setup steps results in a non-styled 18F site. NPM reports "module not found" errors, included below in **Error log**. I'm also seeing that it's overwriting the lockfileVersion from 2 to 1.

![Screenshot 2024-04-11 at 1 53 45 PM](https://github.com/18F/18f.gsa.gov/assets/1887182/d19dcd21-5870-4c44-b85b-cbc92b50670f)

## Steps to reproduce the behavior

With macOS 13.6.4 (Ventura), Ruby 2.3.2, npm 6.13.4, node 10.19.0:

- [ ] Follow the steps in [local development](https://github.com/18F/18f.gsa.gov?tab=readme-ov-file#local-development)
- [ ] Once the site is serving, visit http://0.0.0.0:4000/site

## This issue is done when

- [ ] Visiting the local site shows a correctly styled site
- [ ] NPM version is specified (if important / relevant to the fix)


Error log:

```sh
→  ./serve

> 18f.gsa.gov@1.0.0 uswds-build /Users/username/code/not-project/18f.gsa.gov
> npm run uswds-copy-assets && npm run uswds-compile


> 18f.gsa.gov@1.0.0 uswds-copy-assets /Users/username/code/not-project/18f.gsa.gov
> npx gulp copyAssets

{ Error: Cannot find module 'csso/syntax'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/username/code/not-project/18f.gsa.gov/node_modules/postcss-csso/cjs/index.cjs:3:16)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3) code: 'MODULE_NOT_FOUND' }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! 18f.gsa.gov@1.0.0 uswds-copy-assets: `npx gulp copyAssets`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the 18f.gsa.gov@1.0.0 uswds-copy-assets script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/username/.npm/_logs/2024-04-11T17_52_30_340Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! 18f.gsa.gov@1.0.0 uswds-build: `npm run uswds-copy-assets && npm run uswds-compile`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the 18f.gsa.gov@1.0.0 uswds-build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/username/.npm/_logs/2024-04-11T17_52_30_372Z-debug.log

> 18f.gsa.gov@1.0.0 uswds-watch /Users/username/code/not-project/18f.gsa.gov
> npx gulp watch

{ Error: Cannot find module 'csso/syntax'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/username/code/not-project/18f.gsa.gov/node_modules/postcss-csso/cjs/index.cjs:3:16)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3) code: 'MODULE_NOT_FOUND' }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! 18f.gsa.gov@1.0.0 uswds-watch: `npx gulp watch`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the 18f.gsa.gov@1.0.0 uswds-watch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/username/.npm/_logs/2024-04-11T17_52_31_336Z-debug.log
⚠️   1
```

--- a/package.json
+++ b/package.json
@@ -7,7 +7,8 @@
     "test": "tests"
   },
   "engines": {
-    "node": ">=10.14"
+    "node": ">=18.7",
+    "npm": ">=7.24"
   },
   "scripts": {
     "federalist": "npm run uswds-build && mkdir ./.bundle && echo '---\nBUNDLE_GEMFILE: \"GemfileFederalist\"' > ./.bundle/config",

